src/validation.cpp

```diff
@@ -3278,6 +3324,8 @@ static bool AcceptBlock(const std::shared_ptr<const CBlock>& pblock, CValidation
     if (fCheckForPruning)
         FlushStateToDisk(state, FLUSH_STATE_NONE); // we just allocated more disk space for block files
 
+    CheckBlockIndex(chainparams.GetConsensus());
```

Comment:
This was to replace the one removed from ProcessNewBlock. Should be +/- equivalent to the old version, I believe?